index.html

```diff
@@ -0,0 +1,203 @@
+                    </div>
+                </article>
+                 <article class="project6">
+                    <div class="wrap-card">
+                        <h3>project 6</h3>
+                        <p> Tempore quia aspernatur exercitationem quo libero? Quod cupiditate, minima, asperiores architecto deleniti ullam tenetur doloremque earum nihil quam, maxime quo enim et!</p>
+                        <div class="lang">
+                            <button type="button">html</button>
+                            <button type="button">bootstrap</button>
+                            <button type="button">Ruby</button>
+                        </div>
+                    </div>
+                    <div class="card-link">
+                        <a class="see-project" href="#">See Project</a>
+                    </div>
+                </article> 
+            </section>
+            <section class="aboutme">
+                <h1>About me</h1>
+                <h2>I'm a Software Developer</h2>
+                <p>
+                    Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+                </p>
+                <div class="get-resume">
+                    <a class="see-project" href="#">Get my resume</a>
+                </div>
+            </section>
+            <div class="skills">
+                    <ul id="title1">
+                        <li><img src="/assets/square2.png" alt="squared shapped bullet point">&nbsp; Language &nbsp;&nbsp;&nbsp; </li>
+                        <li class="items">Javascript</li>
+                        <li class="items">Ruby</li>
+                        <li class="items">Html</li>
+                        <li class="items">CSS</li>
+                    </ul>
+            
+                    <ul id="title2">
+                        <li><img src="/assets/square1.png" alt="squared shapped bullet point">&nbsp; Frameworks &nbsp;&nbsp;&nbsp; </li>
+                        <li class="items">Bootstrap</li>
+                        <li class="items">Ruby on Rails</li>
+                        <li class="items">RSPec</li>
+                        <li class="items">CapyBara</li>
+                        <li class="items">Selenium</li>
+                    </ul>
+               
+                    <ul id="title3">
+                        <li><img src="/assets/round.png" alt="squared shapped bullet point">&nbsp; Skills &nbsp;&nbsp;&nbsp; </li>
+                        <li class="items">Codekit</li>
+                        <li class="items">Github</li>
+                        <li class="items">Codepen</li>
+                        <li class="items">Gitlab</li>
+                        <li class="items">Terminal</li>
+                    </ul>
+            </div>
```

Comment:
- [ ] [REQUIRED] Good job so far, please make changes! Your all technology `icons` not display on your project.

![image](https://user-images.githubusercontent.com/63915024/170846542-afdbe509-85b4-4993-bfa1-2ea53d219adc.png)

Reply:
First of all, thanks a lot for your review.
I just checked my results using Live Server, and the images show up pretty fine. But now I checked the actual web address and indeed no image is displayed. I'm trying to find what went wrong, but I've been researching for more than 20 min already. Could you give me a hint?
I checked:
- misspelling
- put full path instead of only the folder and image (e.g. /assets/icon-menu.pg)
- syntax in HTML

why is is working perfectly using Live Server?

![127 0 0 1_5500_index html(iPhone XR)](https://user-images.githubusercontent.com/99205303/170866764-ee742a6e-7b95-40ec-8bbb-b032cd5025e1.png)
![127 0 0 1_5500_index html(iPhone XR) (1)](https://user-images.githubusercontent.com/99205303/170866772-1d6397bb-c8b2-406f-9f0d-3a28871642cd.png)
![127 0 0 1_5500_index html(iPhone XR) (2)](https://user-images.githubusercontent.com/99205303/170866787-c9e88eff-79b7-4045-8284-842c2304fd3c.png)

Reply:
Acctually I've just got it!